keys.go

```diff
@@ -130,24 +137,48 @@ func (c *Client) CreateRootKey(ctx context.Context, name string, expiration *tim
 	return c.CreateKey(ctx, name, expiration, false)
 }
 
+// CreateRootKeyWithSHA1 creates a new, non-extractable key resource without
+// key material using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateRootKeyWithSHA1(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
+	return c.CreateKeyWithSHA1(ctx, name, expiration, false)
+}
+
 // CreateStandardKey creates a new, extractable key resource without
 // key material.
 func (c *Client) CreateStandardKey(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
 	return c.CreateKey(ctx, name, expiration, true)
 }
 
+// CreateStandardKeyWithSHA1 creates a new, extractable key resource without
+// key material using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateStandardKeyWithSHA1(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
+	return c.CreateKeyWithSHA1(ctx, name, expiration, true)
+}
+
 // CreateImportedRootKey creates a new, non-extractable key resource
 // with the given key material.
 func (c *Client) CreateImportedRootKey(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string) (*Key, error) {
 	return c.CreateImportedKey(ctx, name, expiration, payload, encryptedNonce, iv, false)
 }
 
+// CreateImportedRootKeyWithSHA1 creates a new, non-extractable key resource
+// with the given key material and using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateImportedRootKeyWithSHA1(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string) (*Key, error) {
+	return c.CreateImportedKeyWithSHA1(ctx, name, expiration, payload, encryptedNonce, iv, false)
+}
+
 // CreateStandardKey creates a new, extractable key resource with the
 // given key material.
 func (c *Client) CreateImportedStandardKey(ctx context.Context, name string, expiration *time.Time, payload string) (*Key, error) {
 	return c.CreateImportedKey(ctx, name, expiration, payload, "", "", true)
 }
 
+// CreateImportedStandardKeyWithSHA1 creates a new, extractable key resource with the
+// given key material using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateImportedStandardKeyWithSHA1(ctx context.Context, name string, expiration *time.Time, payload string) (*Key, error) {
+	return c.CreateImportedKeyWithSHA1(ctx, name, expiration, payload, "", "", true)
+}
+
```

Comment:
This doesn't need to be a method either, since TKEK import of standard keys is not supported.